runtime/runtime_test.go

```diff
@@ -5475,7 +5446,7 @@ func TestRuntimeContractWriteback(t *testing.T) {
 		},
 		storage: newTestLedger(nil, onWrite),
 		getSigningAccounts: func() ([]Address, error) {
-			return []Address{common.BytesToAddress(addressValue.Bytes())}, nil
+			return []Address{Address(addressValue)}, nil
```

Comment:
Cleanup: No need to convert from array to slice and back to array, can convert between arrays directly